include/cantera/base/ctexceptions.h

```diff
@@ -62,8 +62,12 @@ class CanteraError : public std::exception
 public:
     //! Normal Constructor for the CanteraError base class
     /*!
-     * @param procedure String name for the function within which the error was
-     *             generated.
+     * @param procedure Name of the function within which the error was
+     *             generated. For member functions, this should be written as
+     *             `ClassName::functionName`. For constructors, this should be
```

Comment:
Does the single backtick do something special in doxygen?

Reply:
It's the way to format something as code using a fixed-width font, like in Markdown.

Reply:
I really wish reST used the same syntax. Ah well...